### PR TITLE
Full implementation of edit card use case to card screen

### PR DESCRIPTION
**Pull request by AlbertYS**
Now can edit multiple choice cards

### CODE REVIEW

**Review by JamieJChew — APPROVED** [review]
looks pretty good, we could break up parts of the CardScreen.java class into some helper methods to make it easier to read if that's possible, but otherwise I think it's good.

**Review by aldob02 — APPROVED** [review]
Looks fine. If there's any way to reduce the number of .add() methods I would do that.

File: src/main/java/createCard/CreateCardOutputData.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public CreateCardOutputData(String message, int type, String question, String an
 
         String strType = "";
         if (type == 1) {
-            strType = "Q and A";
+            strType = "Question and Answer";
```
Comment (AndrewNQN) [inline src/main/java/createCard/CreateCardOutputData.java:51]: Thanks for changing this for me

**Review by AndrewNQN — APPROVED** [review]
CardScreen looks decent. I am going to accept so I can test it manually. Other than that, this file needs quite a bit of refactoring so I will be taking on that job.

**Pull request merged**